common/interfaces.go

```diff
@@ -21,13 +21,13 @@ type SwapState interface {
 // It is implemented by *xmrtaker.swapState and *xmrmaker.swapState
 type SwapStateNet interface {
 	HandleProtocolMessage(msg Message) error
-	ID() types.Hash
+	OfferID() types.Hash
```

Comment:
This PR adds more use of the swap ID and the peer ID to our code base. Renaming adds clarity when it is called in places that are working with multiple ID types.